deploy/resources/addons/dashboard/v0.2.0/release.yaml

```diff
@@ -0,0 +1,319 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app: tekton-dashboard
+  name: tekton-dashboard
+  namespace: tekton-pipelines
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: extensions.dashboard.tekton.dev
+spec:
+  group: dashboard.tekton.dev
+  names:
+    categories:
+    - all
+    - tekton-pipelines
+    kind: Extension
+    plural: extensions
+  scope: Namespaced
+  subresources:
+    status: {}
+  version: v1alpha1
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: tekton-dashboard-minimal
+  namespace: tekton-pipelines
+rules:
+- apiGroups:
+  - security.openshift.io
+  resources:
+  - securitycontextconstraints
+  verbs:
+  - use
+- apiGroups:
+  - route.openshift.io
```

Comment:
is permissions on openshift resources necessary in this manifest

Reply:
This file is the same file in the release of the dashboard.  It must be a mistake in the dashboard release file.  I'll open an issue for this.  I don't want to make changes in this file here because it should be corrected in the dashboard.   Is it OK ?

Reply:
Now resolved for the Tekton Dashboard release file, thanks for spotting this 👍

Reply:
The dashboard release files has fixed.  I’ll update.